src/nonlinear.jl

```diff
@@ -663,6 +662,13 @@ function MathProgBase.optimize!(m::PajaritoModel)
         while (time() - start) < m.time_limit
             flush(STDOUT)
             cut_added = false
+            # WARMSTART MIP FROM UPPER BOUND
+            if !any(isnan,m.solution) && !isempty(m.solution)
```

Comment:
``loadproblem!`` isn't the right place for this. ``setwarmstart!`` is usually called after ``loadproblem!``.